src/main/java/io/stargate/sgv2/jsonapi/config/constants/DocumentConstants.java

```diff
@@ -34,6 +34,9 @@ interface Fields {
     // characters, not length limits (nor empty nor "too long" allowed but validated
     // separately)
     Pattern VALID_NAME_PATTERN = Pattern.compile("[a-zA-Z0-9_\\-]*");
+
+    /** Field name pattern in sort and filter clause. */
+    Pattern VALID_FIELD_NAME_PATTERN = Pattern.compile("[a-zA-Z0-9_.\\-]*");
```

Comment:
Let's rename this as something like `VALID_PATH_PATTERN`; existing `VALID_NAME_PATTERN` is for names.